TOPIC_MODEL_MIGRATION_README.md

```diff
@@ -0,0 +1,148 @@
+# Topic Model Meta Migration
+
+This migration adds support for topic modeling metadata and versioning to the Augur database.
```

Comment:
I think it would be nice to include documentation on what topic modeling is and/or why it may be helpful to the augur codebase (and ideally "how" if theres anything special happening there) in addition to just "what" happened.

As someone who hasn't interacted with topic modeling before, I think it would be nice to ensure that the documentation is written with non-experts in mind. It seems to me like the goal of this is to automatically (rather than manually) group items in augur, such as repositories or words from issue/PR comments, into topic categories. It would be great to have your help in understanding whether or not this is correct and making sure the knowledge is documented for future contributors.